### PR TITLE
layers: Correct drm_format_mod_explicit nullptr check

### PR DESCRIPTION
**Pull request by marfre02-ARM**
Change-Id: Ibee54a6675305b653fa9f5227fe9c7477726ee74

### CODE REVIEW

**Comment by marfre02-ARM** [comment]
**Change Note**
Prior to adding drmFormatModifier to the list of format modifiers for VkImage, a check is made on the validity of the pointer to be dereferenced which needs to be corrected.

File: layers/parameter_validation_utils.cpp [hunk 1]
```diff
@@ -1030,7 +1030,7 @@ bool StatelessValidation::manual_PreCallValidateCreateImage(VkDevice device, con
                                      "vkCreateImage(): Tiling is VK_IMAGE_TILING_DRM_FORMAT_MODIFIER_EXT but pNext must have "
                                      "either VkImageDrmFormatModifierListCreateInfoEXT or "
                                      "VkImageDrmFormatModifierExplicitCreateInfoEXT in the pNext chain");
-                } else if (drm_format_mod_list != nullptr) {
+                } else if (drm_format_mod_explict != nullptr) {
```
Comment (ncesario-lunarg) [inline layers/parameter_validation_utils.cpp:1033]: :grimacing: yikes, thanks for catching this!

**Review by ncesario-lunarg — APPROVED** [review]
LGTM! Will merge once internal CI passes.

**Comment by ncesario-lunarg** [comment]
Runing through [internal CI](http://erusea:4040/8466).

**Pull request merged**